README.md

```diff
@@ -77,7 +77,7 @@ GitHub actions is a reasonable choice.
 
 If you only want to (re)generate certain files, you can specify them as arguments to the shell script:
 ```shell
-$TMP/generate.sh README.md coq-action.yml
+$TMP/generate.sh README.md .github/workflows/docker-action.yml
```

Comment:
Did you re-test this?

Otherwise I guess this should be 
```suggestion
$TMP/generate.sh README.md docker-action.yml
```
because the `.yml` file is in the project root folder

Reply:
This is a fix indeed.

If my template repo is in the master branch and I'm in another repo in a sibling directory, `../templates/generate.sh coq-action.yml` does nothing and `../templates/generate.sh .github/workflows/coq-action.yml` does regenerate the file from the template.

Reply:
@Zimmi48 but I'm puzzled to see that apparently, the `generate.sh` script does not reads its arguments(?)

Reply:
Nevermind, I've just seen the `for specified_target in "$@"; do` line, so that's OK…